source/Dockerfile.backend

```diff
@@ -26,8 +26,20 @@ COPY ./ /app/
 COPY --from=dependencies /app /app
 WORKDIR /app/OptimaJet.DWKit.StarterApplication
 RUN dotnet publish -c Release -o out
+# Need dummy values for POSTGRES_* variables for migrations to run
+RUN POSTGRES_HOST=build \
```

Comment:
I don't understand this comment.
what do you mean?

I thought the variables were checked at runtime? in which case you'd want to set them in the deployment environment:
like, 
![image](https://user-images.githubusercontent.com/199018/44477436-adc33200-a608-11e8-9460-ae4c8338c270.png)

in kubernetes.

or like, 
```
docker run --env VAR1=value1 --env VAR2=value2 ubuntu 
```

using vanilla docker

Reply:
`dotnet ef migrations` loads the assembly (using its own main) and dynamically calls `public static IWebHost BuildWebHost(string[] args)` to acquire the DbContext to run the migrations.  This all happens at build-time.

Our code expects to be run only during run-time and requires (via `GetVarOrThrow`) values for POSTGRES_*.

A database connection is not required for generation of the migrations at build-time, so this change provides some dummy values to satisfy our startup code in `Startup.ConfigureServices`.